File: haushaltsbuch/src/main/java/org/github/haschi/haushaltsbuch/api/_ErfasseInventar.java
package org.github.haschi.haushaltsbuch.api;

import org.axonframework.commandhandling.TargetAggregateIdentifier;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung;
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Anweisung;

@Anweisung
public interface _ErfasseInventar
{
    @TargetAggregateIdentifier
    Aggregatkennung für();

    Inventar inventar();
}

File: haushaltsbuch/src/main/kotlin/org/github/haschi/haushaltsbuch/api/ErfasseInventar.kt
package org.github.haschi.haushaltsbuch.api

import org.axonframework.commandhandling.TargetAggregateIdentifier
import org.github.haschi.haushaltsbuch.infrastruktur.modellierung.de.Aggregatkennung

data class ErfasseInventar(
        @TargetAggregateIdentifier val für: Aggregatkennung,
        val inventar: Inventar)
